apps/ledger-live-mobile/src/screens/Platform/v2/Catalog/types.ts

```diff
@@ -0,0 +1,29 @@
+import { TextInput } from "react-native";
+import {
+  BaseComposite,
+  StackNavigatorProps,
+} from "../../../../components/RootNavigator/types/helpers";
+import { DiscoverNavigatorStackParamList } from "../../../../components/RootNavigator/types/DiscoverNavigator";
+import { ScreenName } from "../../../../const";
+
+export type NavigationProps = BaseComposite<
+  StackNavigatorProps<
+    DiscoverNavigatorStackParamList,
+    ScreenName.PlatformCatalog
+  >
+>;
+
+export interface PlatformState {
```

Comment:
Again, I feel like it's outside of Catalog scope. Should it be placed under `../` instead?